Modules/signalmodule.c

```diff
@@ -1663,6 +1663,11 @@ _PySignal_Fini(void)
 #endif
 }
 
+int
+PyErr_HasSignals(void) {
+    return _Py_atomic_load(&is_tripped);
+}
+
```

Comment:
New public functions must be documented in the C-API section.

@vstinner please take a look

Reply:
The latest version of the PR removes `PyErr_HasSignals`.